### PR TITLE
Adding support for setting the topic of a channel

### PR DESCRIPTION
**Pull request by davidsiefert**
I currently use IRC to notify a channel of progress in a playbook, and I would like to sometimes set the topic of the channel as updates progress.  This change allows one to optionally set a "topic" argument to do so.

### CODE REVIEW

File: notification/irc.py [hunk 1]
```diff
@@ -47,6 +47,11 @@
       - The message body.
     required: true
     default: null
+  topic:
```
Comment (bcoca) [inline notification/irc.py:50]: missing version added

Comment (davidsiefert) [inline notification/irc.py:50]: What number would I put here? 1.9?

Comment (davidsiefert) [inline notification/irc.py:50]: Ah, based on recent commit history it looks like it should be 2.0

**Comment by davidsiefert** [comment]
Hey @bcoca added "version-added" to my topic parameter--I'm hoping this is what you were looking for.  Thanks!

**Pull request merged**